bin/migration

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+var mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+var Song = require('../models/song');
+var Language = require('../models/language');
+
+var MongoURI = process.env.MONGOURI || 'mongodb://heroku_z2hck36f:gi10ruk7mcsdedgjkdampa3mh3@ds033153.mlab.com:33153/heroku_z2hck36f';
```

Comment:
Never put security sensitive strings directly in code. Load them from config that is not checked into source control. Put that config file in .gitignore to help protect against this mistake. For now you can get a new key from mlab and update what is in heroku. Ideally, you mongodb config that you use for development could have a prodution file alongside it and that is where you store the secret key.